src/dronevis/detection_torch/yolov5_torch.py

```diff
@@ -15,18 +17,27 @@ class YOLOv5(CVModel):
     For more details see `YOLOv5 <https://pytorch.org/hub/ultralytics_yolov5>`_.
     """
 
+    local_name = "ultralytics_yolov5_master"
+    remote_name = "ultralytics/yolov5"
+    model_local_path = f"/home/{getpass.getuser()}/.cache/torch/hub/{local_name}"
+    model_source = "local"
+    model_name = "yolov5s"
+
     def __init__(self) -> None:
         """Initialize local path"""
         self.net = None
-        self.model_local_path = f"/home/{getpass.getuser()}/.cache/torch/hub/ultralytics_yolov5_master"
 
     def load_model(self) -> None:
         """Load model from PyTorchHub"""
         print("Loading YOLOv5 Torch model ...")
         if os.path.exists(self.model_local_path):
-            self.net = torch.hub.load(self.model_local_path, "yolov5s", source="local")
+            self.net = torch.hub.load(
```

Comment:
since model_local_path is a class attribute, I think you should reference it using the class name instead of self,
and the same goes for the other class variables